test/command/1039.md

```diff
@@ -0,0 +1,34 @@
+```
+% pandoc -f markdown -t markdown-definition_lists
+Term 1
+:   Definition 1.1
+:   Definition 1.2
+
+    Definition 1.2.a
+
+:   Definition 1.3
+
+Term 2
+:   Definition 2.1
+
+    :   Definition 2.1.1
+:   Definition 2.2
+^D
+-   Term 1
+
+    -   Definition 1.1
+
+    -   Definition 1.2
+
+        Definition 1.2.a
+
+    -   Definition 1.3
+
+-   Term 2
+
+    -   -   Definition 2.1
```

Comment:
This is how reader parses this, `Definition 2.1` is parsed as definition, not label. So the result is not what #1039 author expected, but it's correct.